internal/models/task.go

```diff
@@ -38,6 +38,12 @@ type HealthStatus struct {
 
 type TaskStatus struct {
 	Id           string `json:"id,omitempty"`
+	Created      float64 `json:"created,omitempty"`
```

Comment:
These changes make Task and TaskStatus structs almost identical. Maybe we should merge them into one?

Reply:
Yeah, I think we could. Let me work on that

Reply:
I played around with this and I'm not sure it'd be a good thing to join Task and TaskStatus.
We use Task as an entity, for definition of the task. We use TaskStatus for communication between client and server. 
I think it might be okay to leave this as is for now.

Reply:
`TaskStatus` is a misleading name though